version/1/4/mk/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Пакт за Придонесување - Kодекс на Oднесување
```

Comment:
To be honest, I wouldn't even translate it, as it's the name of the project and in my opinion should stay the same in all languages. But since it's translated in all other languages, I guess we should stick with it. "Пакт за Соработник" looks good.